source/state_representation/include/state_representation/MathTools.hpp

```diff
@@ -12,13 +12,6 @@ namespace state_representation::math_tools {
  */
 const Eigen::Quaterniond log(const Eigen::Quaterniond& q);
 
-/**
- * @brief Calculate the exp of a quaternion as a non-unit quaternion
- * @param  q the quaternion to apply the exp on
- * @return the exp of the quaternion
- */
-const Eigen::Quaterniond exp(const Eigen::Quaterniond& q, double lambda = 1);
```

Comment:
Why are you removing the function? Just because we are not using it somewhere else? If we keep the `log`, the `exp` should be there as well.

Reply:
Well the log is actually being used in a few places..

Reply:
I know. So for consistency I would keep the `exp` as well. The function by itself is not wrong. Just the usage of it was not good.